### ISSUE TITLE
Visual constructor and Visual update console.logs unwanted

### ISSUE THREAD

**Issue by Althain** [microsoft/powerbi-client-angular#34]
Hello,

Related to version 3.0.5

I keep seeing in the console two events fired: **Visual constructor** and **Visual update** despite I passed undefined to eventHandlers.

Can I turn off these console logs?

```
<powerbi-report [eventHandlers]="undefined">
```
![image](https://user-images.githubusercontent.com/29130365/220419282-265c122b-8c31-4ad2-9c6b-dd9bb75c567c.png)

**Comment by bapat-atharva** [comment]
Could not reproduce this issue, moreover the logs look like from a visual you are using, whiteBoxData ?

**Comment by Althain** [comment]
Hello, thanks for the response. 

I didn't have direct access to powerBI and I don't really work with it, so I don't know how things work.
I asked my colleague to enter powerBI page and the logs are present there as well, so they come from BI report.  
When I digged in a bit more, I saw those logs on microsoft docs: https://learn.microsoft.com/en-US/power-bi/developer/visuals/highlight?tabs=Standard 
So most likely someone copy-pasted the examples and the code would have to be edited to remove it.

Anyway it has nothing to do with this library. Apologies for the confusion and thanks for your time.
